plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveFileFormats.java

```diff
@@ -616,10 +616,11 @@ public void testParquetPageSourceSchemaEvolution(int rowCount)
     public void testParquetCaseSensitivity(int rowCount)
             throws Exception
     {
-        TestColumn writeColumn = new TestColumn("UPPER_CASE_COLUMN", createVarcharType(4), new HiveVarchar("test", 4), utf8Slice("test"));
-        TestColumn readColumn = new TestColumn("uppeR_casE_columN", createVarcharType(4), new HiveVarchar("test", 4), utf8Slice("test"));
+        TestColumn writeColumnA = new TestColumn("UPPER_CASE_COLUMN", createVarcharType(5), new HiveVarchar("testA", 5), utf8Slice("testA"));
```

Comment:
This test covers the `ParquetTypeUtils` change, not the `PrunedBlockMetadata` change.

Reply:
It covers both, removing change in either place will fail this test